scripts/migrate-exchange-widget.py

```
@@ -0,0 +1,203 @@
+    m = re.search(r'\n<Steps ', content)
+    if m:
+        insert_pos = m.start()
+        return content[:insert_pos] + f"\n\n{widget_tag}\n" + content[insert_pos:]
+
+    # Fallback: insert before inline CTA
+    m = re.search(r'\n<CTA .+variant="secondary"', content)
+    if m:
+        insert_pos = m.start()
+        return content[:insert_pos] + f"\n\n{widget_tag}\n" + content[insert_pos:]
+
+    print(f"  WARNING: No insertion point found")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)